src/m365/pp/commands/dataverse/dataverse-table-row-remove.spec.ts

```diff
@@ -0,0 +1,244 @@
+import * as assert from 'assert';
+import * as sinon from 'sinon';
+import appInsights from '../../../../appInsights';
+import auth from '../../../../Auth';
+import { Cli } from '../../../../cli/Cli';
+import { CommandInfo } from '../../../../cli/CommandInfo';
+import { Logger } from '../../../../cli/Logger';
+import Command, { CommandError } from '../../../../Command';
+import request from '../../../../request';
+import { pid } from '../../../../utils/pid';
+import { powerPlatform } from '../../../../utils/powerPlatform';
+import { sinonUtil } from '../../../../utils/sinonUtil';
+import commands from '../../commands';
+const command: Command = require('./dataverse-table-row-remove');
+
+describe(commands.DATAVERSE_TABLE_ROW_REMOVE, () => {
+  let commandInfo: CommandInfo;
+  //#region Mocked Responses
+  const validEnvironment = '4be50206-9576-4237-8b17-38d8aadfaa36';
+  const validId = '3a081d91-5ea8-40a7-8ac9-abbaa3fcb893';
+  const validTableName = 'DataverseTable';
+  const validEntitySetName = 'cr6c3_dataversetables';
+  const envUrl = "https://contoso-dev.api.crm4.dynamics.com";
+  const tableResponse = {
+    EntitySetName: 'cr6c3_dataversetables'
+  };
+  //#endregion
+
+  let log: string[];
+  let logger: Logger;
+  let promptOptions: any;
+  let loggerLogToStderrSpy: sinon.SinonSpy;
+
+  before(() => {
+    sinon.stub(auth, 'restoreAuth').callsFake(() => Promise.resolve());
+    sinon.stub(appInsights, 'trackEvent').callsFake(() => { });
```

Comment:
Hi, just popping in to notify you that we should update this line to:

```
sinon.stub(telemetry, 'trackEvent').callsFake(() => { });
```

to align it with our latest change: #4215